### DISCUSSION TITLE
Export report data

### DISCUSSION THREAD

**Discussion by thmo** [kimai/kimai#4342] [General]
Maybe I missed it, but is there an option to easily export the report from `.../reporting/user/month` as a excel sheet, csv, or similar?

**Comment by kevinpapst** [comment]
Nope. There should a feature request existing. Please search it and vote via 👍

**Comment by thmo** [comment]
The only feature request I found was this: https://github.com/kimai/kimai/issues/2460 .

**Reply by kevinpapst** [reply]
🤷 then open a new one 😄

**Reply by thmo** [reply]
Done: #4348
